test_analysis6.py

```
@@ -0,0 +1,49 @@
+                        count += 1
+
+                if 'test' not in filepath and 'model' not in filepath and 'entity' not in filepath and 'constants' not in filepath:
+                    sizes.append((filepath, count))
+
+sizes.sort(key=lambda x: x[1], reverse=True)
+
+print("Checking test coverage for all logic files...")
+
+no_tests = []
+for t, s in sizes:
+    if s < 30: continue
```

Ruff (0.15.13)

[error] Multiple statements on one line (colon)

(E701)